lib/Doctrine/Migrations/FileQueryWriter.php

```diff
@@ -42,8 +43,8 @@ public function write(
         string $direction,
         array $queriesByVersion,
         ?DateTimeInterface $now = null
-    ) : bool {
-        $now = $now ?? new DateTimeImmutable();
+    ): bool {
+        $now ??= new DateTimeImmutable();
```

Comment:
is `??=` available for php 7.2?

Reply:
no, it is not. that's a PHP 7.4 feature

Reply:
I should add this to the list of disabled rules